mubench.reviewsite/tests/ReviewControllerTest.php

```diff
@@ -54,4 +54,60 @@ function test_stores_violation_types()
         self::assertEquals("missing/call", $violation_types[0]["name"]);
     }
 
+    function test_determine_next_misuse()
+    {
+        $this->setupSomeRunWithThreeMisusesAndTwoReviews();
+
+        list($previous_misuse, $next_misuse, $next_reviewable_misuse, $misuse)  = $this->reviewController->determineNavigationTargets(Misuse::all(), '-p-', '-v-', '0', Reviewer::find(1));
+        self::assertEquals('2', $previous_misuse->misuse_muid);
```

Comment:
Please save the misuses in variables and use their muids as the expected values here, otherwise one has to trace the ids to check what this means.